packages/dependency-path/src/index.ts

```diff
@@ -143,7 +143,13 @@ export function parse (dependencyPath: string) {
 }
 
 export function depPathToFilename (depPath: string) {
-  const filename = depPathToFilenameUnescaped(depPath).replace(/[\\/:*?"<>|]/g, '+')
+  let filename = depPathToFilenameUnescaped(depPath).replace(/[\\/:*?"<>|]/g, '+')
+  if (filename.includes('(')) {
+    filename = filename
+      .replace(/\)\(/g, '_')
+      .replace(/\(/, '_')
```

Comment:
should this be `.replace(/\(/g, '_')` also? Then this can be simplified to `filename.replace(/(\)\()|\(/g, '_')`